### PR TITLE
[EN/RU] Update `COT22` to Finals

### PR DESCRIPTION
**Pull request by Kimitakari**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Tournaments/COT/2022/en.md [hunk 1]
```diff
@@ -67,6 +67,60 @@ See the [participants sheet](https://docs.google.com/spreadsheets/d/1iie_zXj1GbS
...
+  4. [Marmalade butcher - Amanita (omtereston) \[Collab Extra\]](https://osu.ppy.sh/beatmapsets/1604167#fruits/3349336)
+  5. [Paul Bazooka - DrunkenSteiN (Nao Tomori) \[Collab Extreme\]](https://osu.ppy.sh/beatmapsets/915322#fruits/1925556)
+  6. [Caravan Palace - Lone Digger (Ohai) \[Wii2ZeeBro's Extra\]](https://osu.ppy.sh/beatmapsets/454079#fruits/1165848)
+- Hidden
+  1. [goreshit - broken promises/scattered dreams (Bunnrei) \[team philippines/overdose\]](https://osu.ppy.sh/beatmapsets/1265137#fruits/2629332)
+  2. [SUKIMASWITCH - Golden Time Lover (Cut Ver.) (Crowley) \[The Staircase of Babel\]](https://osu.ppy.sh/beatmapsets/1530609#fruits/3130684)
+  3. [Falcom Sound Team jdk - Seductive Shudder (jonathanlfj) \[Bewitching OWC\]](https://osu.ppy.sh/beatmapsets/1306626#fruits/2709094)
+  4. [sky_delta - Kreuz (Side) \[Irre's Extra\]](https://osu.ppy.sh/beatmapsets/347551#fruits/832346)
+- HardRock
+  1. [*namirin - Koishiteiku Planet (Absolute Zero) \[Team Russia's Overdose\]](https://osu.ppy.sh/beatmapsets/1045892#fruits/2186433)
```
Comment (Walavouchey) [inline wiki/Tournaments/COT/2022/en.md:87]:
```suggestion
  1. [\*namirin - Koishiteiku Planet (Absolute Zero) \[Team Russia's Overdose\]](https://osu.ppy.sh/beatmapsets/1045892#fruits/2186433)
```

File: wiki/Tournaments/COT/2022/ru.md [hunk 2]
```diff
@@ -68,7 +68,61 @@ COT 2022 организован различными членами сообще
...
+  4. [Marmalade butcher - Amanita (omtereston) \[Collab Extra\]](https://osu.ppy.sh/beatmapsets/1604167#fruits/3349336)
+  5. [Paul Bazooka - DrunkenSteiN (Nao Tomori) \[Collab Extreme\]](https://osu.ppy.sh/beatmapsets/915322#fruits/1925556)
+  6. [Caravan Palace - Lone Digger (Ohai) \[Wii2ZeeBro's Extra\]](https://osu.ppy.sh/beatmapsets/454079#fruits/1165848)
+- Hidden
+  1. [goreshit - broken promises/scattered dreams (Bunnrei) \[team philippines/overdose\]](https://osu.ppy.sh/beatmapsets/1265137#fruits/2629332)
+  2. [SUKIMASWITCH - Golden Time Lover (Cut Ver.) (Crowley) \[The Staircase of Babel\]](https://osu.ppy.sh/beatmapsets/1530609#fruits/3130684)
+  3. [Falcom Sound Team jdk - Seductive Shudder (jonathanlfj) \[Bewitching OWC\]](https://osu.ppy.sh/beatmapsets/1306626#fruits/2709094)
+  4. [sky_delta - Kreuz (Side) \[Irre's Extra\]](https://osu.ppy.sh/beatmapsets/347551#fruits/832346)
+- HardRock
+  1. [*namirin - Koishiteiku Planet (Absolute Zero) \[Team Russia's Overdose\]](https://osu.ppy.sh/beatmapsets/1045892#fruits/2186433)
```
Comment (Walavouchey) [inline wiki/Tournaments/COT/2022/ru.md:90]:
```suggestion
  1. [\*namirin - Koishiteiku Planet (Absolute Zero) \[Team Russia's Overdose\]](https://osu.ppy.sh/beatmapsets/1045892#fruits/2186433)
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Comment by Walavouchey** [comment]
@TicClick

File: wiki/Tournaments/COT/2022/ru.md [hunk 3]
```diff
@@ -68,7 +68,61 @@ COT 2022 организован различными членами сообще
 
 Полный список участников есть в гугл-доке в разделе [Игроки](https://docs.google.com/spreadsheets/d/1iie_zXj1GbS_Sx4MyWdGOYD9raofZ46uomMbVW6wm48/edit#gid=749874175).
 
-## Маппулы
+## Mappools
```
Comment (TicClick) [inline wiki/Tournaments/COT/2022/ru.md:71]: вижу копипасту, заголовки здесь и далее нужно перевести на русский

**Review by TicClick — CHANGES_REQUESTED** [review]
looks good to me otherwise

**Review by TicClick — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**